packages/api-page-builder/src/graphql/crud/pageBlocks.crud.ts

```diff
@@ -287,11 +290,18 @@ export const createPageBlocksCrud = (params: CreatePageBlocksCrudParams): PageBl
         ) {
             const blocks = [];
 
-            for (const pageBlock of content?.elements) {
-                const blockId = pageBlock.data?.blockId;
-                // If block has blockId, then it is a reference block, and we need to get elements for it.
+            /**
+             * If the content block has a `blockId`, then it is a referenced block.
```

Comment:
BTW do we need to make similar changes in `packages/api-page-builder/src/graphql/crud/pageTemplates.crud.ts:305` ?

Reply:
No, because template passes block interpolation to `pageBlocks.crud`. The `pageTemplates.crud` is only responsible for loading the template definition, and assigning variable values to the `content.data`. Then it passes actual block content loading to `pageBlocks.crud`. So this logic is not duplicated.